privacyidea/lib/tokenclass.py

```diff
@@ -1643,4 +1643,37 @@ def generate_symmetric_key(self, server_component, client_component,
         key = server_component[:-len(client_component)] + client_component
         return key
 
+    @classmethod
```

Comment:
``@staticmethod`` would be more suitable here, see [here](https://stackoverflow.com/questions/1950414/what-does-classmethod-do-in-this-code/1950927#1950927) and [here](https://stackoverflow.com/questions/136097/what-is-the-difference-between-staticmethod-and-classmethod-in-python)